test/unit/tuple_bigref.result

```diff
@@ -1,17 +1,4140 @@
 	*** main ***
-1..2
-	*** test_bigrefs_overall ***
-    1..3
-    ok 1 - All tuples have refs == 1.
-    ok 2 - All tuples have bigrefs.
-    ok 3 - All tuples were deleted.
-	*** test_bigrefs_overall: done ***
+1..3
+	*** test_one ***
+    1..15
+    ok 1 - allocated
+    ok 2 - no bigrefs
+    ok 3 - Deleting tuple *was* allocated
+    ok 4 - deallocated
+    ok 5 - no bigrefs
+    ok 6 - allocated
+    ok 7 - no bigrefs
+    ok 8 - Deleting tuple *was* allocated
+    ok 9 - deallocated
+    ok 10 - no bigrefs
+    ok 11 - allocated
+    ok 12 - bigrefs
+    ok 13 - Deleting tuple *was* allocated
+    ok 14 - all deallocated
+    ok 15 - no bigrefs
+	*** test_one: done ***
 ok 1 - subtests
-	*** test_bigrefs_non_consistent ***
-    1..3
-    ok 1 - All tuples have bigrefs.
-    ok 2 - 11 tuples don't have bigrefs and all other tuples have
-    ok 3 - All tuples have bigrefs and their indexes are in right order.
-	*** test_bigrefs_non_consistent: done ***
+	*** test_batch ***
+    1..3084
```

Comment:
Please do something about this result file. 4K lines is way too many.

Reply:
fixed.